app/controllers/participate/login.js

```diff
@@ -24,7 +24,7 @@ export default Ember.Controller.extend({
           this.transitionToRoute('participate.survey.consent');
         })
         .catch((e) => {
-          if (e.status === 404) {
+          if (e.status === 404 || e.status === 401) {
```

Comment:
*amendment above: well defined places to deal with internal errors in ember, like transition issues etc. 

It makes sense to handle a network error directly via promise,  but when we start handling totally different kinds of error on the same promise, it gets a bit weirder. The error hook may or may not be useful for, eg, dealing with the failed transition.